web/src/interfaces.ts

```diff
@@ -24,6 +24,29 @@ export interface Image {
   alt?: string;
 }
 
+/**
+ * FontSize type that provides several preset options for a font size.
+ */
+type FontSize = 'small' | 'medium' | 'large';
+
+/**
+ * CommitProgressFontSize type that provides several preset options for the font size of CommitProgress.
```

Comment:
it exceeds 80 chars. can you break this line into two lines?

Reply:
@ifhuang Okay!